public/app/core/components/Footer/Footer.tsx

```diff
@@ -89,3 +106,19 @@ export const Footer: FC = React.memo(() => {
 });
 
 Footer.displayName = 'Footer';
+
+function FooterItem({ item }: { item: FooterLink }) {
+  const content = item.url ? (
+    <a href={item.url} target={item.target} rel="noopener" id={item.id}>
```

Comment:
```suggestion
    <a href={item.url} target={'_blank'} rel="noopener" id={item.id}>
```
Should we just hardcode `target` as `_blank`? Can't really think of a case where it'd be beneficial to have it configurable.

Reply:
good shout. did this!